config/config-mattermod.default.json

```diff
@@ -110,6 +110,12 @@
     "MattermostWebhookURL": "",
     "MattermostWebhookFooter": "",
 
+    "RequestTimeoutInSeconds": 60,
+    "EETaskTimeoutInSeconds": 300,
+    "CronTaskTimeoutInSeconds": 300,
+    "BuildMobileTaskTimeoutInSeconds": 7200,
+    "BuildSpinmintTaskTimeoutInSeconds": 2700,
```

Comment:
I thought as the first cut, we agreed for them to be consts in the code? Unless needed, I would want to avoid more config knobs.

Reply:
Those are optional, we're creating the default when the config is loaded. But this provides flexibility if we need to tune the timeouts in production with just a restart instead of making a deployment. This way we should be able to change timeouts if the default ones are not long enough.

I'll remove the timeouts in the configuration but I'll leave it configurable as a "private" feature right now. What do you think?

Reply:
Ok, but then let's change the fields to be time.Duration instead of int. It feels odd to have getters which convert int to time.Duration every time we call it.

Reply:
Hum but in that case, if we introduce those fields in the configuration file are going to be integers and the JSON parse is going to fail.

Reply:
That's a good point, but isn't too hard to achieve: https://play.golang.org/p/Zm6hpNR-ZJ2.

Reply:
Finally, I've removed the configuration complexity